### PR TITLE
chore: use GitHub URL for source root

### PR DESCRIPTION
**Pull request by jrandolf-google**
`release-please` will update the versions whenever they need updating.

### CODE REVIEW

File: packages/puppeteer-core/package.json [hunk 1]
```diff
@@ -129,13 +129,7 @@
     }
   },
   "files": [
-    "lib",
```
Comment (Lightning00Blade) [inline packages/puppeteer-core/package.json:132]: Revert this change this was reported back that we should include src so that source maps work.
Also the other prevents test from being in the submitted upload.

Comment (jrandolf) [reply packages/puppeteer-core/package.json:132]: It's okay to remove this now. The source map will point back to the GitHub url.

File: packages/puppeteer/package.json [hunk 2]
```diff
@@ -110,13 +110,7 @@
   },
   "files": [
     "lib",
-    "src",
```
Comment (Lightning00Blade) [inline packages/puppeteer/package.json:113]: Same here.
